mne/inverse_sparse/tests/test_mxne_optim.py

```diff
@@ -104,15 +104,15 @@ def test_l21_mxne():
 
 def test_tf_mxne():
     """Test convergence of TF-MxNE solver"""
-    alpha_space = 10.
-    alpha_time = 5.
+    alpha_space = 40.
+    alpha_time = 0.1
```

Comment:
These constants should be renamed to reflect their new names. Also, there are a number of places these constants have been changed. This sounds backward-incompatible to me. Or is it just to improve the quality of the results over what we had before?